wcfsetup/install/files/lib/data/user/User.class.php

```diff
@@ -104,6 +105,11 @@ final class User extends DatabaseObject implements IRouteController, IUserConten
 	 */
 	protected static $userOptions;
 	
+	const REGISTER_ACTIVATION_NONE = 0;
+	const REGISTER_ACTIVATION_USER = 1;
+	const REGISTER_ACTIVATION_ADMIN = 2;
+	const REGISTER_ACTIVATION_USER_AND_ADMIN = 3; // self::REGISTER_ACTIVATION_USER | self::REGISTER_ACTIVATION_ADMIN
```

Comment:
```suggestion
	const REGISTER_ACTIVATION_USER_AND_ADMIN = self::REGISTER_ACTIVATION_USER | self::REGISTER_ACTIVATION_ADMIN
```
This should work with modern PHP versions.